src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -25,6 +32,28 @@ export default function WebexMeetingControl({type, meetingID}) {
     </Button>
   );
 
+  let select;
+
+  if (options) {
+    select = (
+      !compact
+      && (
+        <label htmlFor="dropdown-control">
+          {text}
+          <select id="dropdown-control" onChange={(event) => action(event.target.value)} value={selected}>
```

Comment:
if the `select` is within `label`, the link between them is established without using `for` and `id`. Otherwise, we would have had to generate a unique id here (there might be multiple dropdown controls on the page).